### PR TITLE
present, active voice related changes to SSO main page

### PR DESCRIPTION
**Pull request by Simplychee**
# What changed

https://deploy-preview-1571--logz-docs.netlify.app/user-guide/users/single-sign-on/

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/user-guide/users/single-sign-on/index.md [hunk 1]
```diff
@@ -12,16 +12,16 @@ contributors:
...
+Single sign-on (SSO) lets you manage access to your Logz.io account
 from a single user and identity management tool.
 
 ## What is SSO?
 
 SSO is a way to authenticate and sign in to different services
 with the same credentials.
-With SSO, users won't need to remember
+With SSO, users don't need to remember
 different usernames and passwords for each service,
```
Comment (YajB) [inline _source/user-guide/users/single-sign-on/index.md:23]: For consistency, should this also change from "service" to "**source**"?

File: _source/user-guide/users/single-sign-on/index.md [hunk 2]
```diff
@@ -12,16 +12,16 @@ contributors:
...
-and you can manage access to all your services from one service.
+and you can manage access to all your services from one source.
```
Comment (YajB) [inline _source/user-guide/users/single-sign-on/index.md:24]: Do you mean "data source"?

**Review by YajB — COMMENTED** [review]
Excellent changes!

File: _source/user-guide/users/single-sign-on/index.md (hunk 2 above)
Comment (Simplychee) [reply _source/user-guide/users/single-sign-on/index.md:24]: I meant that they can use one service to handle everything. However, I did use the word service three times in that sentence, and source can also work, so I'm good with this fix.

**Pull request merged**